doc/pundit.md

```diff
@@ -0,0 +1,60 @@
+# Policies
+
+Our policy objects are the heart of the authorizations in Panoptes. Roughtly
+speaking they perform three functions:
+
+1. The `Scope` objects determine which models a user is allowed to access.
+2. The `index?` methods authorize each user for a specific controller action.
+3. The `linkable_xyz` methods return a list of valid Xyzs for a specific model's association.
```

Comment:
perhaps `linkable_relation_name`  vs the xyz one. Also perhaps it could use an explicit example combined with the description.